### ISSUE TITLE
Script not activating

### ISSUE THREAD

**Issue by wallacewinfrey** [Jazqa/kwin-quarter-tiling#61]
I've been using this script for about 6 months and for the most part, few issues. However, the last couple days it has stopped working completely. I'll check the box to activate it, and hit apply, but the windows don't tile (or do anything). If I click away from the KWin Scripts section of systemsettings5 and then back, the checkbox is cleared (which probably explains the lack of tiling). 

Why won't the script stay active, and what can I do to debug it? I've tried removing it and reinstalling it, even switching to the qml and rework branches and trying to activate those, but they won't activate.

**Comment by wallacewinfrey** [comment]
So, appears to be a defect on kwin's part. Every time I would attempt to activate the script, it would result in a copy of kwinrc being created in .config that looks like this:

`-????????? ? ?        ?           ?            ? kwinrc`

There are about 80 of these in there as the result of my trying to figure out what was going on. Unfortunately, I can't seem to figure out how to remove them. I tried listing by inode but to no avail. I was able to move everything but those files out of `.config`, renamed it to`.config.f**ked`, and then move the other files back into a new `.config` directory. Rebooted, new kwinrc file was initialized, now the script activates. However, this is very bizarre behavior, that is almost certainly not the fault of your script, but may be useful for you. Feel free to close at any time.
